modules/react-components/src/components/modal/confirmation-modal/confirmation-modal.tsx

```diff
@@ -159,6 +164,16 @@ export const ConfirmationModal: FunctionComponent<ConfirmationModalPropsInterfac
         onPrimaryActionClick(e);
     };
 
+    /**
+     * Handler for the enter key press down.
+     */
+    const handleKeyDown = (e: KeyboardEvent) => {
+        if (e.key === "Enter" && confirmed) {
+            setAssertionInput("");        
+            onEnterKeyPressConfirm(e);
```

Comment:
Let's trigger the primary action when the `enter` key is pressed. This way we will not have to refactor everywhere where this modal is used.

```suggestion
            onPrimaryActionClick(e);
```